RELEASE-NOTES.md

```diff
@@ -5,7 +5,8 @@
 ### Enhancements
 
 - Added support for handling failed logins due to having an unverified account email [#2976](https://github.com/automattic/simplenote-electron/pull/2976)
-- Add support for handling passwords found in known data breaches [#2972](https://github.com/automattic/simplenote-electron/pull/2972)
+- Added support for handling passwords found in known data breaches [#2972](https://github.com/automattic/simplenote-electron/pull/2972)
+- Updated simperium library dependancy to latest version [#2984](https://github.com/automattic/simplenote-electron/pull/2984)
```

Comment:
typos: "Simperium" should be capitalized and "dependency" is spelled with an E

Reply:
Thank you!